test/functional/JLM_Tests/testng.xml

```diff
@@ -22,7 +22,6 @@
   SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
 -->
 
-<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
```

Comment:
This should stay, but use http, else we get a warning:
```
[TestNGContentHandler] [WARN] It is strongly recommended to add "<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >" at the top of your file, otherwise TestNG may fail or not work as expected.
```
It was only changed to appease xslt. Neither is required in #19062.